mino/minogrpc/server_test.go

```diff
@@ -84,7 +106,75 @@ loop:
 	server.grpcSrv.GracefulStop()
 	err = server.httpSrv.Shutdown(context.Background())
 	require.NoError(t, err)
+}
+
+func Test_ErrorsSimpleCall(t *testing.T) {
+	identifier := "127.0.0.1:2000"
+
+	addr := address{
+		id: identifier,
+	}
+
+	server, err := CreateServer(addr)
+	require.NoError(t, err)
+	server.StartServer()
+
+	handler := testSameHandler{}
+	uri := "blabla"
+	rpc := RPC{
+		handler: handler,
+		srv:     *server,
+		uri:     uri,
+	}
+
+	// Using a wrong request message (nil) should yield an error while decoding
+	respChan, errChan := rpc.Call(nil, &players{players: []address{addr}})
```

Comment:
I guess you can move your players implementation to the test file and call it `fakePlayers`.